packages/devtools_app/lib/src/inspector/flutter/inspector_screen.dart

```diff
@@ -241,7 +242,7 @@ class _InspectorScreenBodyState extends State<InspectorScreenBody>
         inspectorService: inspectorService,
         treeType: FlutterTreeType.widget,
         onExpandCollapseSupported: _onExpandCollapseSupported,
-        onLayoutExplorerSupported: _onLayoutExplorerSupported,
+        onLayoutExplorerSupported: kIsWeb ? null : _onLayoutExplorerSupported,
```

Comment:
it is better to put these sorts of tweaks in their own CL so that they can be undone with a trivial revert.